src/main/java/ca/bc/gov/backendstartapi/config/CorsConfig.java

```diff
@@ -23,7 +22,7 @@ public class CorsConfig implements WebMvcConfigurer {
    */
   @Override
   public void addCorsMappings(CorsRegistry registry) {
-    if (!ObjectUtil.isEmptyOrNull(allowedOrigins)) {
+    if (allowedOrigins.length != 0) {
```

Comment:
I would suggest to keep the older code, as it is null safe

Reply:
Added a null check to the condition.